examples/jsm/loaders/3DMLoader.js

```diff
@@ -124,6 +125,7 @@ class Rhino3dmLoader extends Loader {
 
 					worker._callbacks[ taskID ] = { resolve, reject };
 
+					// TODO if abortSignal is defined, listen to it to cancel the worker
```

Comment:
ideally the abort signal would be handled to "kill" the web worker. having never used web worker I prefer not to try it myself.